Cargo.toml

```
@@ -37,7 +37,7 @@ cobalt-async = { version = "0.4.0", features = ["checksum"]}
 futures = "0.3.25"
 pin-project-lite = "0.2.9"
 serde = { version = "1.0.150", features = ["derive"] }
-serde_json = "1.0.89"
+serde_json =  "1.0.89"
```

```suggestion
serde_json = "1.0.89"
```

This looks like a spurious change, adjusting for consistency.